### PR TITLE
Adds Forked Spear

### PR DESCRIPTION
**Pull request by Dozed12**
Adds a fork spear, slightly better than a wooden spear.

*Now with spikes on the tip.

### CODE REVIEW

**Comment by John-Candlebury** [comment]
I have my doubts about this item, dont really think adding a fork to the tip of a spear would actually make the thing more lethal, as in my experience common dinnereware is actually made of very low quality metal.

Also are you aware of the existence of knife spears?
 They pretty much are the same item, and much easier to craft while at it.

**Comment by KA101** [comment]
Yeah, I think John-Candlebury's comic is on point here.  It's an interesting idea but odds are nobody will ever use this as written.

Now, if it was a trident, infantry fork, or other such implement--I read "fork spear" and thought "a spear with two points on the end", forex--then it'd have a thematic niche and might qualify for DISARM once we get that sorted out.

Up for a redesign?

**Comment by Zireael07** [comment]
+1 for trident idea.

**Comment by Dozed12** [comment]
I like that idea I will take care asap.

**Comment by Zireael07** [comment]
Change name, please. Fork spear doesn't say anything about it. "Trident" would be better.

**Comment by Dozed12** [comment]
Well, KA101 said that when he read fork spear he thought about a trident... but maybe trident spear ?

**Comment by narc0tiq** [comment]
Add me to the people for whom "fork spear" sounds like "fork on a stick".

**Comment by kevingranade** [comment]
Association with knife spear, which is a knife (actually several) on the
end of a stick makes it problematic.
'Forked spear' might also be ok if it's just a two-pronged spear.

File: data/json/items/melee.json [hunk 1]
```diff
@@ -4591,6 +4591,23 @@
...
+    {
+        "type":"GENERIC",
+        "id" : "spear_forked",
+        "name" : "forked spear",
+        "description" : "A wooden spear with a trident like tip. ",
+        "weight" : 1100,
+        "to_hit" : 1,
+        "color" : "brown",
+        "symbol" : "/",
+        "material" : ["wood", "null", "steel"],
```
Comment (KA101) [inline data/json/items/melee.json]: I'm not sure why null's needed here.  Cutting the proportions?

**Comment by KA101** [comment]
Here are some examples of fork-like polearm heads.  I'm thinking this unit is pretty much the one on the far left: http://nethackwiki.com/mediawiki/images/thumb/2/2f/Forks.jpg/400px-Forks.jpg

**Comment by Dozed12** [comment]
I made it more like the second from the right ( recipient uses 3 spikes), about the Null, I used the wooden spear as an example and changed accordingly, should I take null out?

**Comment by Rivet-the-Zombie** [comment]
>should I take null out?

Yes.

**Comment by Dozed12** [comment]
Done.

**Comment by KA101** [comment]
OK.  Couple things to keep in mind for future (I'll take care of 'em here):
1) Tabs are the enemy, as "tab" characters parse differently from system to system.  If you're on Windows, Notepad++ has an option in the language settings to have your Tab key insert spaces instead of a tab.  Four spaces is the DDA standard indent.

2) Try to insert new gear near other gear like it, not at the end of the list.  People adding different stuff to the end of the same list is a common cause of merge conflicts: they're usually not difficult to resolve (unless in vehicles--those can be a PAIN), but they can get annoying.

Thanks.

**Comment by narc0tiq** [comment]
@KA101 
> "tab" characters parse differently from system to system

Please don't lie -- they **parse** exactly the same in every system (as a Tab character -- 0x09). What _is_ different in every editor is how wide a Tab character is -- some people have it set to 2 spaces, some to 4, etc.. This introduces a maintenance headache and makes it harder for people to read the code, and therefore harder to contribute.

Otherwise, entirely agree with everything you said.

**Comment by Dozed12** [comment]
Thank you, I will keep in mind that.

**Comment by KA101** [comment]
>Please don't lie

Then I misspoke, and I regret the error.  That was not an intentional attempt to mislead, thanks much.

**Pull request merged**